### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by CsharpNoobDude** [Nostrademous/Dota2-FullOverwrite#203]
If we do not see the enemy carry in the lane - except heroes like riki, pa, who can be invisible - we should send a support to ward the jungle - or maybe even block a camp with a sentry / ward.
If they see the hero in the jungle, they should try to gank him. If low enough hp, then the support may do it alone, but else the mid (Invoker) and the offlaner should help. Team decision stuff.
Just to make sure the enemy carry does not get free money. 
About the blocking camps - if we see an enemy stacking camps, we should make sure to kill that camp - or if it is too strong block it, and try to kill the bigger creeps at least.

**Comment by Nostrademous** [comment]
I agree with all of this, easier said than done :P but yes, ultimately the plan is to write all the code to do that.

**Comment by CsharpNoobDude** [comment]
Yes, I know - but they are only suggestions. You can leave this "issue" pinned here.
I think it is kinda weird, that there are only issues that you can post. No suggestions, no comments...
